games-fps/alephone/alephone-9999.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools optfeature xdg
+
+DESCRIPTION="An enhanced version of the game engine from the classic Mac game, Marathon"
+HOMEPAGE="http://source.bungie.org/"
+if [[ ${PV} = 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Aleph-One-Marathon/alephone/"
+	EGIT_SUBMODULES=() # Upstream includes game data as submodules, we only want the engine
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/Aleph-One-Marathon/alephone/archive/refs/tags/release-${PV}.tar.gz"
+	S="${WORKDIR}/${PN}-release-${PV}"
+fi
+
+LICENSE="GPL-3+ BitstreamVera OFL-1.1"
+SLOT="0"
+
+IUSE="alsa curl speex upnp"
+
+# ffmpeg covers most audio/video use cases and the package doesn't configure with alternatives enabled (media-libs/smpeg)
+# When resolved upstream, !ffmpeg ( media-libs/libmad media-libs/libsndfile media-libs/libvorbis media-libs/smpeg )
+# with an appropriate REQUIRED_USE may be added.
+# See https://github.com/Aleph-One-Marathon/alephone/issues/85
+RDEPEND="
+	dev-libs/boost:=
+	dev-libs/expat
+	dev-libs/zziplib:=
+	media-libs/libpng
+	media-libs/libsdl2
+	media-libs/sdl2-image[png]
+	media-libs/sdl2-net
+	media-libs/sdl2-ttf
+	media-video/ffmpeg:=
+	sys-libs/zlib
+	virtual/opengl
+	virtual/glu
+	alsa? ( media-libs/alsa-lib )
+	curl? ( net-misc/curl )
+	speex? ( media-libs/speex )
+	upnp? ( net-libs/miniupnpc )
+"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+
+DEPEND="
+	${RDEPEND}
+"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--enable-lua \
+		--enable-opengl \
+		--with-ffmpeg \
+		--without-mad \
+		--without-smpeg \
+		--without-sndfile \
+		--without-vorbis \
+		$(use_with alsa) \
+		$(use_with curl) \
+		$(use_with speex) \
+		$(use_with upnp miniupnpc)
+}
+
+src_install() {
+	default
+	dobin "${FILESDIR}"/${PN}.sh
+	doman docs/${PN}.6
+	docinto html/
+	dodoc docs/*.html
+}
+
+pkg_preinst() {
```

Comment:
You can drop this bit :) (the phase is exported and nothing else is providing it (no other eclasses), so it gets called anyway)